src/consts/values.ts

```diff
@@ -1,3 +1,7 @@
 export const MIN_ROUNDED_VALUE = 0.00001;
 export const DISPLAY_DECIMALS = 5;
 export const STANDARD_TOKEN_DECIMALS = 18;
+export const SOL_ZERO_ADDRESS = '00000000000000000000000000000000000000000000';
```

Comment:
fwiw the byte `0` is encoded in base58 is `1`, so this is actually not a valid Solana base58 address. But `111111...1111` is taken as the SystemProgram's address, so as long as we don't ever try to treat this const as valid base58 and just as a marker for the native token internal to the codebase, we shouldn't have issues